ui/src/app/shared/services/workflows-service.ts

```diff
@@ -32,7 +32,11 @@ export class WorkflowsService {
     }
 
     public watch(filter: {namespace?: string; name?: string; phases?: Array<string>; labels?: Array<string>}): Observable<models.kubernetes.WatchEvent<Workflow>> {
-        const url = `api/v1/workflow-events/${filter.namespace || ''}?${this.queryParams(filter).join('&')}`;
+        const params = this.queryParams(filter);
+        params.push(
+            `fields=result.object.metadata.name,result.object.metadata.namespace,result.object.status.finishedAt,result.object.status.startedAt,result.object.status.phase`
+        );
+        const url = `api/v1/workflow-events/${filter.namespace || ''}?${params.join('&')}`;
```

Comment:
This seems to be fixing the `workflow-events` endpoint issue. Shouldn't this be done in #3165 instead?